.github/actions/build/action.yml

```diff
@@ -23,6 +23,6 @@ runs:
         mkdir build && cd build && cmake .. && msbuild diskann.sln /m /nologo /t:Build /p:Configuration="Release" /property:Platform="x64" -consoleloggerparameters:"ErrorsOnly;Summary"
         cd ..
         mkdir dist
-        mv .\x64\Release\ .\dist\bin
+        mklink /j .\dist\bin .\x64\Release\
```

Comment:
what does this do?  is this why I couldn't get the exes to run from a different directory?  teach me! :D